contributing_to_docs/create_or_edit_content.adoc

```diff
@@ -168,11 +165,32 @@ any further changes that are required, and finally merge your work.
 your feature branch that includes all of your work.
 2. Click on *Pull Request* to submit the PR against the `main` branch of the
 `openshift-docs` repository.
-3. If you know which product versions your change applies to, include a comment
+3. Fill out the information requested on the template.
+** If you know which product versions your change applies to, include a comment
 that specifies the minimum version that the change applies to. The docs team
 maintains these branches for all active and future distros and your PR will be
 applied to one or more of these branches.
-4. Tag the documentation team with @openshift/team-documentation (if you are a part of the OpenShift organization. If not, tag @vikram-redhat).
+*** PR applies to all versions after a specific version (e.g. 4.8): 4.8+
+*** PR applies to the in-development version (e.g. 4.12) and future versions: 4.12+
+*** PR applies only to a specific single version (e.g. 4.10): 4.10
+*** PR applies to multiple specific versions (e.g. 4.8-4.10): 4.8, 4.9, 4.10
+** Provide a link to the preview. Automatic preview functionality is currently only available for some branches. For PRs that update the rendered build in any way against branches that do not create an automated preview:
+*** OpenShift documentation team members (core and aligned) must include a link to a locally generated preview.
+*** External contributors can request a generated preview from the OpenShift documentation team.
+** All documentation changes that impact the meaning of the docs must be verified by a QE team associate before merging.
+** Provide any other information about the change that the docs team might need to understand it.
+4. Make sure that you squash to one commit before submitting your PR.
+5. Ask for review from the OpenShift docs team:
+** For community authors: Request a review by tagging @openshift/team-documentation or @kalexand-rh in a GitHub comment.
+** For Red Hat associates:
+*** For normal peer requests, add a comment that contains this text: /label peer-review-needed
+*** For normal merge review requests, add a comment that contains this text: /label merge-review-needed
+*** For urgent peer review requests, ping @peer-review-squad requesting a review in the #forum-docs-review channel (CoreOS Slack workspace) and provide the following information:
+**** A link to the PR.
+**** The size of the PR that the GitHub bot assigns (ex: XS, S, M, L, XL).
+**** Details about how the PR is urgent.
+*** For urgent merge requests, ping @merge-review-squad in the #forum-docs-review channel (CoreOS Slack workspace).
+*** Except for changes that do not impact the meaning of the content, QE review is required before content is merged.
```

Comment:
You did already mention that QE is required in step 3, not sure if you wanted to repeat it again here or not

Reply:
I think it was mentioned twice in the original content. The context is different enough (and the statement is important enough) that I think it's ok to say twice.

Reply:
Works for me! A reminder isn't a bad thing :wink: